src/Pages/Auth/Register/Register.tsx

```diff
@@ -1,10 +1,165 @@
-import React from 'react'
-import './Register.module.scss'
-
+import React from "react";
+import "./Register.module.scss";
+import { background4 } from "@/Assets/Images";
+import {
+  FaAddressCard,
+  FaCheckCircle,
+  FaEnvelope,
+  FaKey,
+  FaUserPlus,
+  FaUserTie,
+} from "react-icons/fa";
+import { FaRegCircleXmark } from "react-icons/fa6";
+import { Link } from "react-router-dom";
 const Register = () => {
-  return <>
-    <div>Register</div>
-  </>
-}
+  return (
+    <>
+      <div className="bg-mainBg h-screen">
+        <div className="container mx-auto p-5">
+          {/* Top-left Logo */}
+          <div className="text-white my-3 w-40 flex flex-row p-2">
+            <div className="flex">
+              <FaRegCircleXmark className="text-3xl" />
+              <FaCheckCircle className="text-3xl" />
+            </div>
+            <div className="text-lg">| Quizwiz</div>
+          </div>
+         {/* ---------------- */}
+          <div className="grid grid-cols-2 gap-8">
+            <div>
+              {/* header */}
+              <h2 className="text-mainColor font-bold">
+                Create your account and start using QuizWiz!
+              </h2>
+              {/* ------- */}
+
+              {/* Sign Up&In Icons */}
+              <div className="grid grid-cols-4 mt-3">
+                <Link to="/">
+                  <div className="bg-stone-700 me-3 rounded-lg text-center py-3 ">
+                    <FaUserTie className=" text-white text-6xl m-auto" />
+                    <span className="text-white">Sign in</span>
+                  </div>
+                </Link>
+                <div>
+                  <div className=" bg-stone-700 ms-3 rounded-lg text-center py-3 border-4 border-[#C5D86D]  ">
```

Comment:
Why should wrap these elements inside a Link component as the sign-in anchor button.